### PR TITLE
Add missing jsonnet binary to builds

### PR DESCRIPTION
**Pull request by kakkoyun**
This PR fixes Jsonnet issues in makefile.

cc @squat @cben

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,8 +18,9 @@ UP_BIN=$(BIN_DIR)/up
 MEMCACHED_BIN=$(BIN_DIR)/memched
 PROMETHEUS_BIN=$(BIN_DIR)/prometheus
 GOJSONTOYAML_BIN=$(BIN_DIR)/gojsontoyaml
+JSONNET?=$(BIN_DIR)/jsonnet
 # We need jsonnet on CI; here we default to the user's installed jsonnet binary; if nothing is installed, then install go-jsonnet.
-JSONNET_BIN=$(if $(shell which jsonnet 2>/dev/null),$(shell which jsonnet 2>/dev/null),$(BIN_DIR)/jsonnet)
+JSONNET_BIN=$(if $(shell which jsonnet 2>/dev/null),$(shell which jsonnet 2>/dev/null),$(JSONNET))
```
Comment (squat) [inline Makefile]: to have this be a bit more consistent, let's keep the other one as $(JSONNET_BIN) make this special variable something explicit like $(JSONNET_LOCAL_OR_INSTALLED)

**Comment by kakkoyun** [comment]
friendly ping @squat

File: Makefile [hunk 2]
```diff
@@ -201,6 +202,9 @@ $(PROMETHEUS_BIN): $(BIN_DIR)
 $(EMBEDMD_BIN): $(BIN_DIR)
 	GO111MODULE=on go build -mod=vendor -o $@ github.com/campoy/embedmd
 
+$(JSONNET): $(BIN_DIR)
```
Comment (squat) [inline Makefile]: This should be `$(JSONNET_BIN)` right?

Comment (kakkoyun) [reply Makefile]: Yes, thanks for catching that.

**Comment by cben** [comment]
Ack, this works, and the var rename made the makefile clearer :+1:

**Pull request merged**